src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -1,1058 +1,545 @@
 ({
-    /********************************************* View Modules *********************************************/
+    /******************************** Init Functions *****************************/
+    init: function (component) {
```

Comment:
The spaces between `function` and `(component)` are kind of bugging me.

Reply:
Removed.